Directory.Build.props

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <DefaultVersion>1.7.1</DefaultVersion>
```

Comment:
This would be updated when you are ready to release to nuget and other package managers.